homeassistant/components/wallbox/coordinator.py

```diff
@@ -160,6 +165,21 @@ def _get_data(self) -> dict[str, Any]:
         data[CHARGER_STATUS_DESCRIPTION_KEY] = CHARGER_STATUS.get(
             data[CHARGER_STATUS_ID_KEY], ChargerStatus.UNKNOWN
         )
+
+        # Set current solar charging mode
+        eco_smart_enabled = data[CHARGER_DATA_KEY][CHARGER_ECO_SMART_KEY][
+            CHARGER_ECO_SMART_STATUS_KEY
+        ]
+        eco_smart_mode = data[CHARGER_DATA_KEY][CHARGER_ECO_SMART_KEY][
+            CHARGER_ECO_SMART_MODE_KEY
+        ]
+        if eco_smart_enabled is False:
+            data[CHARGER_SOLAR_CHARGING_MODE] = EcoSmartMode.OFF
+        elif eco_smart_mode == 0:
+            data[CHARGER_SOLAR_CHARGING_MODE] = EcoSmartMode.ECO_MODE
+        elif eco_smart_mode == 1:
+            data[CHARGER_SOLAR_CHARGING_MODE] = EcoSmartMode.FULL_SOLAR
```

Comment:
```suggestion
            data[CHARGER_ECO_SMART_KEY] = EcoSmartMode.OFF
        elif eco_smart_mode == 0:
            data[CHARGER_ECO_SMART_KEY] = EcoSmartMode.ECO_MODE
        elif eco_smart_mode == 1:
            data[CHARGER_ECO_SMART_KEY] = EcoSmartMode.FULL_SOLAR
```
can we use the same key here? this makes it consistent with the other keys